### DISCUSSION TITLE
二级域名到端口，怎么配置？

### DISCUSSION THREAD

**Discussion by ChuanV** [jeessy2/ddns-go#1135] [Q&A] [answered]
就比如我要配置ddns.example.com到example.com:9876服务，如何操作？

**Comment by WaterLemons2k** [comment, marked as answer]
Nginx 反代

**Comment by Separatee** [comment]
我是这么解决的。

1.如果你的域名已经备案，直接添加一个隐性url转发，填写地址直接带端口
把example.com隐性转发到example.com:9876

2.如果你的域名没有备案,请先把他DNS换到Cloudflare,这是必须的,或者你可以解析到别的国外的解析商。具体手动搜索一下**换DNS解析商到cloudflare**，不再赘述。

按照ddns-go的webui创建一个api，新建一个全新的dns解析,例如

域名: `ipv6.example.com`
绑定到: `2x8xx0:a7d9:15ec:cxxxx`

同样地，这个就是主要ddns-go要使用的切换地址

**注意**:填写时不带端口，直接绑定你的地址(v4或者v6)
为你想要的域名(例如one.example.com)创建一个CNAME,
绑定到刚刚你的新建域名，如绑定到ipv6.example.com
在Cloudflare菜单中点开
规则---👉Origin rules
新建规则填写如下:


````
字段: 主机名

运算符: 等于

值: one.example.com
````
则:
````
重写到: # 填写你的端口
````

一个基本的步骤就这样完成了。对于其他端口也如此

好处是便于管理，你只需要使用ddns-go管理一个地址ipv6.example.com即可。
